### ISSUE TITLE
Roborock S6 MaxV  | Model does not support the water mode

### ISSUE THREAD

**Issue by spitzboob** [homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum#233]
For the new S6 MaxV model the plugin logs 

`roborock.vacuum.a10 |Model does not support the water mode `
`getSerialNumber | roborock.vacuum.a10 | Serial Number is R0402S01501145`

let me know if you need futher information

**Comment by afharo** [comment]
Nice! New model! It should be pretty simple: just adding your model to https://github.com/homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum/blob/master/models/models.js

Can you try in your local installation to edit the `models/models.js` file and adding the snippet below right after line containing `"roborock.vacuum.a08": [{ speed: speedmodes.gen3 }],`?

```
  "roborock.vacuum.a10": [
    { speed: speedmodes["gen4+custom"], waterspeed: watermodes.gen1 },
  ],
```

If that works, we can easily do the changes in the repo and publish a version with it :)

**Comment by spitzboob** [comment]
@afharo Thanks, works like a charm :)

```
ACT setWaterSpeed | roborock.vacuum.a10 | WaterBoxMode set to 203 over miIO for "Hight".
ACT setWaterSpeed | roborock.vacuum.a10 | WaterBoxMode set to 202 over miIO for "Medium".
ACT setWaterSpeed | roborock.vacuum.a10 | WaterBoxMode set to 201 over miIO for "Light"
ACT setWaterSpeed | roborock.vacuum.a10 | WaterBoxMode set to 200 over miIO for "Off".
```

**Comment by afharo** [comment]
Brilliant! Thanks for confirming! So it's an easy fix we can add it to the next release (I've been a bit dormant with doing changes to this library, sorry)

**Comment by afharo** [comment]
It should be fixed by the version [0.12.0](https://github.com/homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum/releases/tag/v0.12.0)
